examples/apps/collaborative-textarea/webpack.config.js

```diff
@@ -39,14 +39,16 @@ module.exports = env => {
             headers: {
                 'Access-Control-Allow-Origin': '*'
             },
-            publicPath: '/dist',
-            before: (app, server) => fluidRoute.before(app, server, env),
-            after: (app, server) => fluidRoute.after(app, server, __dirname, env),
-            watchOptions: {
-                ignored: "**/node_modules/**",
-            }
+        },
+        // This impacts which files are watched by the dev server (and likely by webpack if watch is true).
```

Comment:
i guess this might explain all the weird file watcher issues we sometimes got with tests

Reply:
That seems likely. This ignore for node_modules does work (but its not where the docs says its supposed to go), but the tests try and override this to avoid watching everything else and that seems to not make a difference. Once we are done with the webpack 5 migration, maybe we can fix it properly.